pyaedt/hfss.py

```diff
@@ -5224,3 +5224,26 @@ def get_antenna_ffd_solution_data(
             overwrite=overwrite,
             taper=taper,
         )
+
+    @pyaedt_function_handler()
+    def set_material_threshold(self, threshold=None):
+        """Set material threshold.
+
+        Parameters
+        ----------
+        threshold : float, optional
+            Conductivity threshold.
+            If "None" default value is 100000.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
+        try:
+            if not threshold:
+                threshold = 100000
+            self.odesign.SetSolveInsideThreshold(threshold)
+            return True
+        except:
+            return False
```

Comment:
I might be missing a point, but why not using `100000` as the default value instead of `None`.
```suggestion
    def set_material_threshold(self, threshold=100000):
        """Set material conductivity threshold.

        Parameters
        ----------
        threshold : float, optional
            Conductivity threshold.
            The default value is 100000.

        Returns
        -------
        bool
            ``True`` when successful, ``False`` when failed.
        """
        try:
            self.odesign.SetSolveInsideThreshold(threshold)
            return True
        except:
            return False
```